### PR TITLE
Blog: Summer launch week hackathon with the PXCI stack

### PR DESCRIPTION
**Pull request by CezzaineZ**
## Summary

Our July 2024 launch week hackathon blog in collaboration with Prisma, Clerk & Inngest.

Blog copy doc: https://docs.google.com/document/d/1CwS5yKdvAuOy-4faWMXDytIzvlS26yQhwuDa2SawCXM/edit
Hackathon planning doc: https://docs.google.com/document/d/1G9u6_jRljhv6ax1JuCsFfECnDv6wX7jk0l_fdtB5v7o/edit#heading=h.5794os9c5kz7

Preview: https://xata-blog-pr-280.vercel.app/blog/summer-launch-pxci-hackathon

### Blog Release Checklist

- \[ ] Release on: \<07/05/24>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: xata-summer-hackathon-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- **Format:** Individual or 2-person teams.
+- **Requirements:** Use the full stack (Xata, Clerk, Inngest, Prisma) to create an app.
+- **Public repositories:** Your code must be public.
+- **New submissions only:** No recycling old projects.
+- **Submission deadline:** July 17, 2024 (midnight local time)
+
+## ❓ How to enter
+
+1. Sign up for a [free Xata account](https://app.xata.io/signin?mode=signup).
+1. Download or sign up for [Prisma](https://console.prisma.io/login), [Inngest](https://app.inngest.com/sign-up), [Clerk](https://dashboard.clerk.com/sign-up).
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:32]:
```suggestion
1. Free install or sign up: [Prisma](https://www.prisma.io/orm), [Inngest](https://app.inngest.com/sign-up), [Clerk](https://dashboard.clerk.com/sign-up).
```

File: xata-summer-hackathon-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Over the past year, we’ve noticed that many of our customers were using the PXCI (Prisma, Xata, Clerk and Inngest) stack and thought it would be fun to host a hackathon for our joint community. As part of our upcoming Xata Launch Week in July 2024, we’re bringing you a two-week coding challenge like no other. Whether you're a solo developer or part of a duo, this is your chance to showcase your creative skills, collaborate with a vibrant community, and win fantastic prizes!
+
+## ✨ Hackathon overview
+
+Excited? We are too! Here are the rules ✅
+
+- **Duration:** July 5 - July 17, 2024
+- **Format:** Individual or 2-person teams.
+- **Requirements:** Use the full stack (Xata, Clerk, Inngest, Prisma) to create an app.
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:24]:
```suggestion
- **Requirements:** Use the free tier of the full stack (Xata, Clerk, Inngest, Prisma) to create an app.
```

**Review by CezzaineZ — COMMENTED** [review]
Prisma amends

**Review by CezzaineZ — COMMENTED** [review]
Add in Clerk changes

File: xata-summer-hackathon-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+- **Public repositories:** Your code must be public.
+- **New submissions only:** No recycling old projects.
+- **Submission deadline:** July 17, 2024 (midnight local time)
+
+## ❓ How to enter
+
+1. Sign up for a [free Xata account](https://app.xata.io/signin?mode=signup).
+1. Free install or sign up: [Prisma](https://www.prisma.io/orm), [Inngest](https://app.inngest.com/sign-up), [Clerk](https://dashboard.clerk.com/sign-up).
+1. Check out this [PXCI starter stack](https://github.com/inngest/next-pxci-starter) to help get you started.
+1. [Submit your project](https://5i8caik7lja.typeform.com/to/tWtK3ZlG) and agree to the [terms and conditions](https://docs.google.com/document/d/1VQCbns0abAt5jBi1e2zf11PMfDMA3MgZw4CjiNkYDUY/edit).
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:34]:
```suggestion
1. [Submit your project](https://xata.io/challenge) and agree to the [terms and conditions](https://docs.google.com/document/d/1VQCbns0abAt5jBi1e2zf11PMfDMA3MgZw4CjiNkYDUY/edit).
```

File: xata-summer-hackathon-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+title: Summer launch week hackathon with the PXCI stack
+description: 'Join our summer hackathon challenge in collaboration with Prisma, Clerk and Inngest to win some amazing prizes!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/holiday-hackathon.png
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:5]:
```suggestion
  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pxci-hackaton@2x.jpg
```

File: xata-summer-hackathon-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  alt: Summer Hackathon
+author: Cezzaine Zaher
+date: 05-07-2024
+tags: ['community', 'hackathon']
+published: true
+slug: summer-launch-pxci-hackathon
+---
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:12]:
```suggestion
ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pxci-hackaton@2x.jpg
---
```

**Review by CezzaineZ — COMMENTED** [review]
adding OGimage

File: xata-summer-hackathon-2024.mdx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+tags: ['community', 'hackathon']
+published: true
+slug: summer-launch-pxci-hackathon
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pxci-hackathon@2x.jpg
+---
+
+We’re excited to invite you to join our summer hackathon in collaboration with our friends at [Prisma](https://www.prisma.io/), [Clerk](https://clerk.com/) and [Inngest](https://www.inngest.com/)!
+
+Over the past year, we’ve noticed that many of our customers were using the PXCI (Prisma, Xata, Clerk and Inngest) stack and thought it would be fun to host a hackathon for our joint community. As part of our upcoming Xata Launch Week in July 2024, we’re bringing you a two-week coding challenge like no other. Whether you're a solo developer or part of a duo, this is your chance to showcase your creative skills, collaborate with a vibrant community, and win fantastic prizes!
+
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:18]:
```suggestion
![Summer Hackathon](/images/pxci-hackathon@2x.jpg)
```

File: xata-summer-hackathon-2024.mdx [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+- **2nd place:** $1000 USD + Xata & Partner Swag
+- **3rd place:** $500 USD + Xata & Partner Swag
+
+## 📅 Key Dates
+
+- **Hackathon kickoff:** July 5, 2024
+- **Submission deadline:** July 17, 2024 (midnight ET)
+- **Winners announcement:** July 19, 2024 at 12:00pm EST live on Discord
+
+## 🆘 Support
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:73]:
```suggestion
## 🆘 Support and resources
```

File: xata-summer-hackathon-2024.mdx [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+Join our dedicated Hackathon channel (**#hackathon**) on the [Xata Community Discord](https://xata.io/discord) for help and support. Xata and Partner teams will be available **Monday to Friday**, from **5am to 5pm ET**. Any queries outside these hours will be addressed to the best of our ability or the next working day.
+
```
Comment (CezzaineZ) [inline xata-summer-hackathon-2024.mdx:76]:
```suggestion

Here's a list of resources you may find useful

- [Xata Summer Hackathon resource hub](https://xata.notion.site/Summer-Hackathon-Resources-a612dffc333a41c2bf9f3f8ed7712750)
- [Clerk Hackathon Resources](https://www.notion.so/Clerk-Hackathon-Resources-1993bf4a3b3841fb91b01b209b9258d1?pvs=21)
- [Inngest Quickstart]( https://www.inngest.com/docs/quick-start)
- Inngest Blog: [AI in production: Managing capacity with flow control](https://www.inngest.com/blog/ai-in-production-managing-capacity-with-flow-control)
```

**Review by CezzaineZ — COMMENTED** [review]
Adding resources section

**Review by tsg — APPROVED** [review]

**Pull request merged**